packages/fetchai/connections/p2p_libp2p/aea/dhtpeer/dhtpeer.go

```diff
@@ -0,0 +1,818 @@
+/* -*- coding: utf-8 -*-
+* ------------------------------------------------------------------------------
+*
+*   Copyright 2018-2019 Fetch.AI Limited
+*
+*   Licensed under the Apache License, Version 2.0 (the "License");
+*   you may not use this file except in compliance with the License.
+*   You may obtain a copy of the License at
+*
+*       http://www.apache.org/licenses/LICENSE-2.0
+*
+*   Unless required by applicable law or agreed to in writing, software
+*   distributed under the License is distributed on an "AS IS" BASIS,
+*   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*   See the License for the specific language governing permissions and
+*   limitations under the License.
+*
+* ------------------------------------------------------------------------------
+ */
```

Comment:
In the python files we use the convention that there is a docstring on the top of each file (one-liner) which describes the module: `"""Module to do xyz"""`. We should replicate this here in all files.

Reply:
yes, definitely.
Proper go documentation is something I still need to look into